basic-rest-apis/rest-api-project/controllers/dashboard.js

```diff
@@ -0,0 +1,30 @@
+
+const express = require('express')
+const User = require("../models/user")
+
+
+exports.userInfo = (req,res,next) =>{
+
+ User.find()
```

Comment:
Add try catch blocks for better exception handling

Reply:
updated